cmd/dlv/cmds/commands.go

```diff
@@ -358,6 +358,10 @@ func execute(attachPid int, processArgs []string, conf *config.Config) int {
 
 	var status int
 	if Headless {
+		if Addr == "localhost:0" {
```

Comment:
I think this would be cleaner if we did something like `RootCmd.Flags().Get("listen").HasChanged()` to indicate that we're only printing if user defined value for that flag, and not relying on our default value, which may change, and it would be non obvious to update this piece of code.